### PR TITLE
export csv for users and teams

### PR DESCRIPTION
**Pull request by tranhuyhoangbka**

### CODE REVIEW

File: app/models/team.rb [hunk 1]
```diff
@@ -10,4 +10,17 @@ class Team < ActiveRecord::Base
...
+    CSV.generate do |csv|
+      csv << Settings.csv.team_column
+      csv << [self.name, self.description, self.leader.name]
+      csv << Settings.csv.member
+      csv << self.users.pluck(:name)
+    end
+  end
+    
+  def leader
+    User.find_by_id self.leader_id
```
Comment (hieuns) [inline app/models/team.rb]: find_by_id bị deprecated ở Rails 4, cậu nên dùng find_by hoặc find

File: app/models/team.rb [hunk 2]
```diff
@@ -10,4 +10,17 @@ class Team < ActiveRecord::Base
   after_create :log_create
   after_update :log_update
   after_destroy :log_delete
+
+  def to_csv
+    CSV.generate do |csv|
+      csv << Settings.csv.team_column
+      csv << [self.name, self.description, self.leader.name]
```
Comment (cuongnguyen2503) [inline app/models/team.rb]: Should remove self anywhere you can.

File: app/models/user.rb [hunk 3]
```diff
@@ -28,4 +28,13 @@ class User < ActiveRecord::Base
   Settings.models.user.roles.each do |role_key, role_value|
     define_method("is_#{role_key}?") {role == role_value}
   end
+
+  def to_csv
+    CSV.generate do |csv|
+      csv << Settings.csv.user_column
+      csv << [self.name, self.email, self.birthday, self.position.name]
```
Comment (cuongnguyen2503) [inline app/models/user.rb]: Should remove self anywhere you can.

File: app/models/team.rb [hunk 4]
```diff
@@ -10,4 +10,17 @@ class Team < ActiveRecord::Base
...
+  def to_csv
+    CSV.generate do |csv|
+      csv << Settings.csv.team_column
+      csv << [name, description, leader.name]
+      csv << Settings.csv.member
+      csv << users.pluck(:name)
+    end
+  end
+    
+  def leader
```
Comment (dieunb) [inline app/models/team.rb]: why don't `team.leader`?

**Pull request merged**